code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -390,7 +409,7 @@
 	T.MakeSlippery(min_wet_time = 10, wet_time_to_add = reac_volume*2)
 	var/obj/effect/hotspot/hotspot = (locate(/obj/effect/hotspot) in T)
 	if(hotspot)
-		var/datum/gas_mixture/lowertemp = T.remove_air(T.air.total_moles())
+		var/datum/gas_mixture/lowertemp = T.remove_air( T:air:total_moles() )
```

Comment:
:eyes:

Reply:
:eyes:

Reply:
Will revert when I get home. This must be something which broke when I copied this from the old repo as I would not have changed this 😂